File: 11364 Parking/solution.java
import java.io.*;

public class Main {
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		StreamTokenizer st = new StreamTokenizer(br);
		st.nextToken();
		int t = (int)st.nval;
		StringBuilder output = new StringBuilder();
		while(t-- > 0) {
		    st.nextToken();
		    int n = (int)st.nval, min = 99, max = 0;
		    while(n-- > 0) {
		        st.nextToken();
		        min = Math.min(min, (int)st.nval);
		        max = Math.max(max, (int)st.nval);
		    }
		    output.append(max - min << 1).append('\n');
		}
		System.out.print(output);
	}
}

File: 11364 Parking/solution.c
#include <stdio.h>

int main() {
	int t;
	scanf("%d", &t);
	while(t--) {
	    int n;
	    scanf("%d", &n);
	    int position, min = 99, max = 0;
	    while(n--) {
	        scanf("%d", &position);
	        min = min < position ? min : position;
	        max = max > position ? max : position;
	    }
	    printf("%d\n", max - min << 1);
	}
	return 0;
}
